week-1/Homework/mandatory/1-syntax-errors.js

```diff
@@ -1,17 +1,19 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
-function addNumbers(a b c) {
+function addNumbers(a ,b ,c) {
     return a + b + c;
 }
 
-function introduceMe(name, age)
-return "Hello, my name is " + name "and I am " age + "years old";
+function introduceMe(name, age){
+return "Hello, my name is" +name 
+ "and I am" +age "years old";
+}
 
 function getRemainder(a, b) {
-    remainder = a %% b;
+    remainder = a % b;
```

Comment:
Hi Tas, on this line you have a variable declaration:
```suggestion
    remainder = a % b;
```
Can you spot what is missing from your variable declaration at the moment ?